inference-engine/ie_bridges/java/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2020 Intel Corporation
+
+find_package(Java REQUIRED)
+include(UseJava)
+
+set(JAVA_AWT_INCLUDE_PATH NotNeeded)
+
+project(inference_engine_java_api)
+
+set(InferenceEngine_LIBRARIES inference_engine)
+
+find_package(JNI REQUIRED)
+
+file(GLOB_RECURSE sources cpp/*.cpp)
+add_library(${PROJECT_NAME} SHARED ${sources})
+
+file(GLOB_RECURSE java_source org/intel/openvino/*.java)
+add_jar(inference_engine_jar ${java_source} 
+        OUTPUT_NAME ${PROJECT_NAME} 
+        OUTPUT_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
+
+target_link_libraries(${PROJECT_NAME} PRIVATE ${InferenceEngine_LIBRARIES})
+target_include_directories(${PROJECT_NAME} PRIVATE ${JNI_INCLUDE_DIRS})
+
+if(ENABLE_TESTS)
+    file(DOWNLOAD https://search.maven.org/remotecontent?filepath=junit/junit/4.13/junit-4.13.jar 
+                  ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/junit-4.13.jar)
+    file(DOWNLOAD https://search.maven.org/remotecontent?filepath=org/hamcrest/hamcrest-core/1.3/hamcrest-core-1.3.jar 
+                  ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/hamcrest-core-1.3.jar)
+
+    file(GLOB_RECURSE java_tests_src tests/*.java)
+    add_jar(ie_java_api_tests_jar ${java_tests_src} 
+            OUTPUT_NAME ie_java_api_tests 
+            OUTPUT_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY} 
+            INCLUDE_JARS ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/*)
+endif()
+
+if(ENABLE_SAMPLES) 
+    find_package(OpenCV) 
+    if (OpenCV_FOUND AND EXISTS "${OpenCV_INSTALL_PATH}/share/java")
```

Comment:
is `OpenCV_INSTALL_PATH` available if I built OpenCV locally with Java?

Reply:
You can install OpenCV locally as well :)
@likholat, what is location of OpenCV's `.jar` in the build folder before `make install`?

Reply:
> You can install OpenCV locally as well :)

I don't want to install, I want to work with build tree directly.

Reply:
Got it, thanks! Let's try to find  flexible way to work with both install and build structures.